ios/FluentUI/ActivityIndicator/ActivityIndicatorTokens.swift

```diff
@@ -13,74 +13,67 @@ import UIKit
     case medium
     case large
     case xLarge
-
-    var size: CGFloat {
-        switch self {
-        case .xSmall:
-            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.xSmall
-        case .small:
-            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.small
-        case .medium:
-            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.medium
-        case .large:
-            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.large
-        case .xLarge:
-            return FluentUIThemeManager.S.MSFActivityIndicatorTokens.size.xLarge
-        }
-    }
 }
 
-/// Representation of design tokens to buttons at runtime which interfaces with the Design Token System auto-generated code.
-/// Updating these properties causes the SwiftUI button to update its view automatically.
-class MSFActivityIndicatorTokens: MSFTokensBase, ObservableObject {
-    @Published public var activityIndicatorSize: CGFloat!
-    @Published public var defaultColor: UIColor!
-    @Published public var thickness: CGFloat!
-
-    var size: MSFActivityIndicatorSize {
-        didSet {
-            if oldValue != size {
-                updateForCurrentTheme()
-            }
-        }
-    }
+/// Design token set for the `ActivityIndicator` control.
+public class ActivityIndicatorTokens: ControlTokens {
+    /// Creates an instance of `ActivityIndicatorTokens` with optional token value overrides.
+    /// - Parameters:
+    ///   - size: MSFActivityIndicatorSize enumeration value that will define pre-defined values for side and thickness.
+    ///   - defaultColor: The default color of the Activity Indicator.
+    ///   - side: The value for the side of the square frame of an Activity Indicator.
+    ///   - thickness: The value for the thickness of the ActivityIndicator ring.
+    public init(size: MSFActivityIndicatorSize,
+                defaultColor: DynamicColor? = nil,
+                side: CGFloat? = nil,
```

Comment:
Took me a second to determine what `side` meant. I know we can't reuse `size` so maybe something related to `dimension` would be more descriptive?

Reply:
diameter?

Reply:
Just copying the same response I posted in the other comment:

`The reasoning behind the name is that for rectangular frames we have different values for height and width. The term in geometry for the height and width of a square is "side" which is why I chose to use it. It implies that the shape of the frame is a square of equal sides.`